transformer_engine/pytorch/module/linear.py

```diff
@@ -1536,25 +1536,11 @@ def _get_weight_tensors(self) -> List[Union[torch.Tensor, QuantizedTensorStorage
     def _get_weight_and_bias_tensors(self) -> Tuple[torch.Tensor, Optional[torch.Tensor]]:
         # Get concatenated weight and bias tensors
         unfused_weights = self._get_weight_tensors()
-        if any(isinstance(w, QuantizedTensor) for w in unfused_weights):
-            if self.fp8:
-                if len(unfused_weights) != 1:
-                    raise RuntimeError(
-                        "Splitting QuantizedTensor into multiple params is not supported"
-                    )
-            else:
-                warnings.warn(
```

Comment:
This was added by @negvet in order to warn the users that they are running non-FP8 workload with quantized params, it is a good check to have. Do we have this elsewhere?